.github/scripts/filter_test_configs.py

```diff
@@ -124,7 +124,7 @@ def get_pr_info(pr_number: int) -> dict[str, Any]:
     )
 
     if not json_response:
-        warnings.warn(f"Failed to get the labels for #{pr_number}")
+        warnings.warn(f"Failed to get the labels for #{pr_number}", stacklevel=2)
```

Comment:
need to exclude .github and scripts as well

Reply:
I was not sure because the .github was not excluded before. But it's reasonable to exclude it.